learn/pkg/learn/renderer.go

```diff
@@ -180,6 +181,50 @@ func (s *evySource) RenderHTML(buf *bytes.Buffer) {
 	buf.WriteString("</code></pre>\n")
 }
 
+func newTxtarContent(filename string, resultType ResultType) (Renderer, error) {
+	b, err := os.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	archive := txtar.Parse(b)
+	m := map[string]bool{}
+	for _, file := range archive.Files {
+		content := string(file.Data)
+		if m[content] {
+			return nil, fmt.Errorf("%w: found duplicate content", ErrBadTxtar)
```

Comment:
can you put `file.Name` in this error message so they know which is one of the duplicates?

Reply:
done, great idea.